example/src/screens/additionalExamples/YodaScreen.js

```diff
@@ -0,0 +1,176 @@
+import React from 'react'
```

Comment:
Example component from readme added to the /example directory - It can be used to quickly test custom props in TabbarHeader